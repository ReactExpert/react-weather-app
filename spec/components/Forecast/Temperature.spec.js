import React from 'react';
import Temperature from '../../../src/components/Forecast/Temperature';
import { enzymeConfig, mount } from '../../enzymeConfig';

enzymeConfig();

describe('Temperature', () => {
  it('matches snapshot', () => {
    const component = mount(<Temperature temperature={22} />);

    expect(component).toMatchSnapshot();
  });
});
