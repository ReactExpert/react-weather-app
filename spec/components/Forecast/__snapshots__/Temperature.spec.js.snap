// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Temperature matches snapshot 1`] = `
<ForeCastTemperature
  temperature={22}
>
  <p
    className="forecast__temperature"
  >
    22
    <span
      className="temperature__unit-dot"
    />
  </p>
</ForeCastTemperature>
`;
